### ISSUE TITLE
Full screen doens't scale the video to full screen

### ISSUE THREAD

**Issue by Yoruba** [mderrick/react-html5video#86]
Is it possible to scale the video to fit into the screen when the player is in full screen mode?
With the HTML5 player the video is fill the whole screen when it is in full screen mode.
(see [link ](https://www.w3schools.com/html/html5_video.asp)for html5 player)

Thanks for the player, Adri

**Comment by mderrick** [comment]
Yes this is how it used to work but you lose the custom styled controls when doing this as per issue https://github.com/mderrick/react-html5video/issues/72. It's either go totally full screen or "half" full screen with custom controls bar. I may consider adding an option to maybe pick the fullscreen mode if you do not mind losing the custom controls when doing so.

**Comment by Yoruba** [comment]
Thanks for you response. 
Maybe the width of the class _2jFFg can be set to the width of the browser?
Then you can keep the controls.

**Comment by cgbrucezjy** [comment]
yes, I would really love to have actual full screen size :)

**Comment by cgbrucezjy** [comment]
also just to share my quick fix.
go into the css file inside react-html5video node mudule and find the css file
update this part with width and height worked for me.
(I did have a div wrap around my video component in normal size so it wont go full screen all the time)
.rh5v-DefaultPlayer_component {
    position: relative;
    font-family: Helvetica;
    font-size: 11px;
    background-color: #000;
    _width:100%;
    height:100%;_
}

**Comment by vipullimbachiya** [comment]
How do I get fullscreen with browser controls? I need scaled video in fullscreen, does not matter if it shows browser controls in the fullscreen. Please help. Thanks.
